### PR TITLE
Zos flist

### PR DESCRIPTION
**Pull request by zaibon**

### CODE REVIEW

File: JumpscaleLib/sal_zos/vm/ZOS_VM.py [hunk 1]
```diff
@@ -223,6 +223,37 @@ def add(self, name, path, content):
         self._items.append(config)
         return config
 
+class KernelArg:
+    def __init__(self, name, key, value=''):
```
Comment (zaibon) [inline JumpscaleLib/sal_zos/vm/ZOS_VM.py]: In this case the key can be the name cause you cannot have two entry with same key but different value

Comment (serboctor) [reply JumpscaleLib/sal_zos/vm/ZOS_VM.py]: That was my initial idea, but when I checked with Jo he told me it is possible and it depends on the consumer of the args.

Comment (zaibon) [reply JumpscaleLib/sal_zos/vm/ZOS_VM.py]: ha ? ok then. all good to go

**Comment by zaibon** [comment]
LGTM, just codacy is not happy. Let's fix these 2 small issue and good to merge

**Pull request merged**